site/sfguides/src/native-app-chairlift/native-app-chairlift.md

```diff
@@ -686,25 +592,12 @@ In this Quickstart, the **Configuration** tab is included to demonstrate how dif
 ## Clean up
 Duration: 1
 
-Let's clean up your Snowflake account. Run the following SQL commands in a worksheet:
+Let's clean up your Snowflake account. In the same terminal you opened before, execute the following command:
 
-```sql
------ CLEAN UP -----
-USE ROLE chairlift_admin;
-DROP APPLICATION chairlift_app;
-DROP DATABASE chairlift_consumer_data;
-
-USE ROLE chairlift_provider;
-DROP APPLICATION PACKAGE chairlift_pkg;
-
-USE ROLE ACCOUNTADMIN;
-DROP WAREHOUSE chairlift_wh;
-DROP ROLE chairlift_provider;
-DROP ROLE chairlift_viewer;
-DROP ROLE chairlift_admin;
```

Comment:
@sfc-gh-osalazarlizano can you re-add all the other lines other than `drop application` and `use role chairlift_provider; drop application package`? We still need to clean up those other things as they won't be dropped by `snow app teardown`.